samples/Dantooine/Dantooine.Client/Client/Shared/NavMenu.razor

```diff
@@ -0,0 +1,49 @@
+﻿<div class="top-row pl-4 navbar navbar-dark">
+    <a class="navbar-brand" href="">Blazor OpenIddict BFF Cookies</a>
```

Comment:
Consider using a shorter name like `Dantooine WASM` or `Dantooine client` to avoid overflows:

![image](https://user-images.githubusercontent.com/6998306/148462236-49ca8a72-3fe7-4bcc-94c1-f76d7f81aa4c.png)

Reply:
Dantooine WASM client?

Reply:
Dantooine client is better

Reply:
updated